File: Ejercicios/MovieRental/MovieRental.ClassLibrary/Movie.cs
namespace MovieRental.ClassLibrary
{
	public enum PriceCodes
	{
		Regular,
		NewRelease,
		Childrens
	}
	public class Movie
	{
		public Movie(string title, PriceCodes priceCode)
		{
			this.Title = title;
			this.PriceCode = priceCode;
		}

        public PriceCodes PriceCode { get; set; }

	    public string Title { get; private set; }
	}
}

File: Ejercicios/Java/MovieRental/Before/src/main/movierental/Movie.java
package movierental;

public class Movie
{
    private PriceCodes priceCode;

    private String title;

    public Movie(String title, PriceCodes priceCode)
    {
        this.title = title;
        this.setPriceCode(priceCode);
    }

	public PriceCodes getPriceCode() {
		return priceCode;
	}

	public void setPriceCode(PriceCodes priceCode) {
		this.priceCode = priceCode;
	}
}